project-chip/connectedhomeip
# Change parameter in Controller local Generate NOC function

#### Problem
What is being fixed?  Examples:
* For implementing concurrent NOC, SDK has to generate Keypair CSR.
* To solve this structural problem, Request to change the parameters of the function that creates the Controller NOC.

#### Change overview
Adding function for generating controller local NOC.
Change parameter (publicKey -> KeyPair)

#### Testing
How was this tested? (at least one bullet point required)
* Check Commissioning Complete
* Check NOC, CSR value.
